ui/js/controllers/perf/graphs.js

```diff
@@ -339,8 +339,11 @@ perf.controller('GraphsCtrl', [
                                     series.projectName, rev).then(
                                         function(resultSets) {
                                             var resultSetId = resultSets[0].id;
-                                            var j = series.flotSeries.resultSetData.indexOf(resultSetId);
-                                            series.highlightedPoints.push(j);
+                                            var j =  _.compact(_.map(series.flotSeries.resultSetData,
+                                                function(resultSet, index) {
+                                                    return resultSet == resultSetId ? index : null;
+                                                }));
+                                            series.highlightedPoints = _.union(series.highlightedPoints, j);
```

Comment:
I would also remove the temporary variable and just append the result of _.compact(...) directly.

E.g.:
```
series.highlightedPoints = _.union(series.highlightedPoints, _.compact(...));
```